tests/api_tests/test_site_hierarchy_api.py

```diff
@@ -0,0 +1,161 @@
+            response,
+            msg="Response structure does not match the expected structure",
+        )
+        self.assertEqual(
+            expected_response_keys, response_data_keys, msg="Items do not match"
+        )
+
+    def test_get_hierarchy_with_awardee_param(self):
+        awardee_id = "PITT"
+        expected_response = _make_expected_response(awardee_id=awardee_id)
+        response = self.send_get(f"SiteHierarchy?awardee_id={awardee_id}")
+        self.assertEqual(
+            expected_response,
+            response,
+            msg="Response structure does not match expected structure",
+        )
+
+    def test_get_hierarchy_with_google_group_param(self):
+        google_group = "hpo-site-clinic-phoenix"
+        expected_response = _make_expected_response(google_group=google_group)
+        response = self.send_get(f"SiteHierarchy?google_group={google_group}")
+        self.assertEqual(
+            expected_response,
+            response,
+            msg="Response structure does not match expected structure",
+        )
+
+    def test_get_hierarchy_with_organization_param(self):
+        organization_id = "PITT_BANNER_HEALTH"
+        expected_response = _make_expected_response(organization_id=organization_id)
+        response = self.send_get(f"SiteHierarchy?organization_id={organization_id}")
+        self.assertEqual(
+            expected_response,
+            response,
+            msg="Response structure does not match expected structure",
+        )
+
+    def test_get_hierarchy_with_multiple_params(self):
+        organization_id, google_group = "PITT_BANNER_HEALTH", "hpo-site-monroeville"
+        expected_response = _make_expected_response(
+            organization_id=organization_id, google_group=google_group
+        )
+        response = self.send_get(
+            f"SiteHierarchy?organization_id={organization_id}&google_group={google_group}"
+        )
+        self.assertEqual(
+            expected_response,
+            response,
+            msg="Response structure does not match expected structure",
+        )
+
+    def test_get_empty_response_with_obsolete_site(self):
+        google_group = "illinois-site-2"
+        expected_response = _make_expected_response(google_group=google_group)
+        response = self.send_get(f"SiteHierarchy?google_group={google_group}")
+        self.assertEqual(
+            expected_response,
+            response,
+            msg="Response structure does not match expected structure. Response should be empty.",
+        )
+
+    def test_empty_response_no_records(self):
+        awardee_id = "no_awardee"
+        expected_response = _make_expected_response(awardee_id=awardee_id)
+        response = self.send_get(f"SiteHierarchy?awardee_id={awardee_id}")
+        self.assertEqual(
+            expected_response,
+            response,
+            msg="Response structure does not match expected structure. Response should be empty.",
+        )
```

Comment:
Can we also add a test for the validation that was included in the API, for testing against a bad key being sent in the params?